src/apps/index.ts

```diff
@@ -7,6 +7,7 @@ export const allowedApps = [
   'sign-message',
   'fear-and-greed',
   'pillar-swap',
+  'basic-transaction',
```

Comment:
Just a reminder we'll swap this out soon for the API endpoint

Reply:
We need a ticket to plug the current API, I thought we had one, but seems no 🤔

Reply:
let me get that sorted now that we have capacity